autospec/buildreq.py

```diff
@@ -304,8 +304,8 @@ def rakefile(filename):
 
 def parse_cmake(filename):
     """Scan a .cmake or CMakeLists.txt file for what's it's actually looking for."""
-    findpackage = re.compile(r"find_package\((\w+)\b.*\)", re.I)
-    pkgconfig = re.compile(r"pkg_check_modules\s*\(\w+ (.*)\)", re.I)
+    findpackage = re.compile(r"^[^#]*find_package\((\w+)\b.*\)", re.I)
```

Comment:
Should we catch spaces before the `#` ?

Reply:
Yes.

Reply:
Actually this regexp is already correct. It accepts any prefix that doesn't contain the comment character.

Reply:
So it already rejects `    # find_package...`, because it contains a comment char in the prefix.